GRIB2_Template_4_144_ProductDefinitionTemplate_en.csv

```diff
@@ -0,0 +1,39 @@
+Title_en,OctetNo,OctetCount,Contents_en,Note_en,noteIDs,codeTable,flagTable,Status
+Analysis or forecast at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for waves selected by period range,10,1,Parameter category,(see Code table 4.1),,4.1,,Operational
+Analysis or forecast at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for waves selected by period range,11,1,Parameter number,(see Code table 4.2),,4.2,,Operational
+Analysis or forecast at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for waves selected by period range,12,1,Type of wave period interval,(see Code table 4.91),,4.91,,Operational
+Analysis or forecast at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for waves selected by period range,13,1,Scale factor of lower wave period limit,,,,,Operational
+Analysis or forecast at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for waves selected by period range,14-17,4,Scaled value of lower wave period limit,(see Note 1),163,,,Operational
+Analysis or forecast at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for waves selected by period range,18,1,Scale factor of upper wave period limit,,,,,Operational
+Analysis or forecast at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for waves selected by period range,19-22,4,Scaled value of upper wave period limit,(see Note 1),163,,,Operational
```

Comment:
Is note 163 correct??
"163,In relation to local coordinate axes at a (grid) cell edge."